README.md

```diff
@@ -94,15 +94,17 @@
 - [Nick Jones](http://www.narrowdesign.com/)
 - [Samuel Ramos](http://www.highfivebro.com/)
 - [Rick Waalders](http://rickwaalders.com/)
+- [Jean-Baptiste Giffard](http://www.digitaldesigner.cool/)
 
 ## Creative Agencies
 - [Bluegg](https://bluegg.co.uk/)
 - [Humaan](https://humaan.com/)
 - [Creative Navy](https://www.interface-design.co.uk/)
 - [Artists Web](https://www.artistsweb.cz/)
-- [Strange Love](http://pupcreativeagency.com/)
+- [Strange Love](http://strangelove.co/)
```

Comment:
dead link

Reply:
Yeah looks like they have recently shut down or not renewed their domain. Unless they changed it to - https://strangelove.nl/ which I don't think can make this list.